flowforge/materials/Fluid.py

```diff
@@ -47,6 +49,13 @@ def viscosity(self, h):
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
I'm thinking maybe this should not be abstract but just raise the `NotImplementedError`

Reply:
This would mean that you would not define surface temperature for the Helium material and if it ever gets called, it would stop.

Reply:
Done

Reply:
Done